installer/roles/kubernetes/templates/deployment.yml.j2

```diff
@@ -281,72 +159,25 @@ spec:
 {% if task_cpu_limit is defined %}
               cpu: "{{ task_cpu_limit }}m"
 {% endif %}
-        - name: {{ kubernetes_deployment_name }}-rabbit
-          image: "{{ kubernetes_rabbitmq_image }}:{{ kubernetes_rabbitmq_version }}"
+        - name: {{ kubernetes_deployment_name }}-redis
+          image: {{ kubernetes_redis_image }}:{{ kubernetes_redis_image_tag }}
           imagePullPolicy: Always
           ports:
-            - name: http
+            - name: redis
               protocol: TCP
-              containerPort: 15672
-            - name: amqp
-              protocol: TCP
-              containerPort: 5672
-          livenessProbe:
-            exec:
-              command:
-              - /usr/local/bin/healthchecks/rabbit_health_node.py
```

Comment:
Is this `rabbit_health_node.py` a thing we wrote?  Is there a file to delete somewhere?

Reply:
Nevermind, I didn't read far enough.  Ignore me.

Reply:
This was mounted via a configmap (another blob deleted further down)

Reply:
Yep, I see it.  Should've looked closer before opening my mouth 😂